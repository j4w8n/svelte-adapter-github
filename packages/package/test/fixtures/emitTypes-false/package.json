{
	"name": "emit-types",
	"private": true,
	"version": "1.0.0",
	"description": "emitTypes settings disabled",
	"type": "module"
}
