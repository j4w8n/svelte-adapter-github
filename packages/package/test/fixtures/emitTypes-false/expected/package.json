{
	"name": "emit-types",
	"private": true,
	"version": "1.0.0",
	"description": "emitTypes settings disabled",
	"type": "module",
	"exports": {
		"./package.json": "./package.json",
		"./Test.svelte": "./Test.svelte",
		"./Test2.svelte": "./Test2.svelte",
		".": "./index.js"
	},
	"svelte": "./index.js"
}